docs/pages/reference/join-methods.mdx

```diff
@@ -440,6 +440,19 @@ Support for self-hosted Terraform Enterprise requires Teleport Enterprise.
 - [Run the Teleport Terraform Provider on Terraform Cloud](../admin-guides/infrastructure-as-code/terraform-provider/terraform-cloud.mdx)
 </Admonition>
 
+### Spacelift: `spacelift`
+
+This join method is used to authenticate using Spacelift. It is typically used
+by the Teleport Terraform provider on either Spacelift cloud or self-hosted
+Spacelift. It can not be used to join Terraform runs on other platforms and
```

Comment:
```suggestion
by the Teleport Terraform provider on Spacelift (including self-hosted
deployments). It can not be used to join Terraform runs on other platforms and
```
"Spacelift cloud" should be a proper noun, but since this doesn't seem to be an official Spacelift edition (it looks like cloud is the default and "self-hosted" is a possibility for Enterprise accounts), I've suggested rephrasing.